docs/running-on-yarn.md

```diff
@@ -479,12 +479,12 @@ Hadoop services issue *hadoop tokens* to grant access to the services and data.
 Clients must first acquire tokens for the services they will access and pass them along with their
 application as it is launched in the YARN cluster.
 
-For a Spark application to interact with HDFS, HBase and Hive, it must acquire the relevant tokens
+For a Spark application to interact with Hadoop filesystem, HBase and Hive, it must acquire the relevant tokens
```

Comment:
I think perhaps we should say interact with any of the Hadoop filesystems (for example hdfs, webhdfs, etc).